dashboard/lib/api/v1/school_district_autocomplete.rb

```diff
@@ -0,0 +1,38 @@
+class Api::V1::SchoolDistrictAutocomplete < AutocompleteHelper
+  # Queries the schools lookup table for schools that match the user-defined
+  # search criteria.
+  # @param query [String] the user-define query string
+  # @param limit [int] the maximum number of results to return
+  # @return [Array] an array of JSON formatted schools
+  def self.get_matches(query, limit)
+    limit = format_limit(limit)
+    query = format_query(query)
+
+    return [] if query.length < MIN_WORD_LENGTH + 2
```

Comment:
Why is this different from the MIN_QUERY_LENGTH in school_autocomplete?

Reply:
We can probably remove MIN_QUERY_LENGTH from school_autocomplete, I just didn't want to mess with that existing code much.